store/store.go

```diff
@@ -1304,7 +1314,9 @@ func (s *Store) Download(ctx context.Context, name string, targetPath string, do
 		// We revert to normal downloads if there is any error.
 		logger.Noticef("Cannot download or apply deltas for %s: %v", name, err)
 	}
-	w, err := os.OpenFile(targetPath+".partial", os.O_RDWR|os.O_CREATE, 0644)
+
+	partialFileName := targetPath + ".partial"
```

Comment:
s/FileName/Path/; we're just appending to targetPath, so that's probably more in line with local code conventions.